hooks/react-client/useSiteNavigation.tsx

```diff
@@ -1,5 +1,7 @@
 'use client';
 
-import useBaseSiteNavigation from '@/hooks/useBaseSiteNavigation';
+const useSiteNavigation = () => {
+  throw new Error('Attempted to call useSiteNavigation from RCC');
```

Comment:
nit/question: should this be RSC?
```suggestion
  throw new Error('Attempted to call useSiteNavigation from RSC');
```

Reply:
Nope, RCC. (React Client Component)